src/components/Navbar.tsx

```diff
@@ -52,6 +52,11 @@ export const Navbar = () => {
             label: "Perks",
             Icon: RxStar,
         },
+        [paths.location]: {
+            label: "Location",
+            Icon: FiMapPin,
+            externalLink: "https://www.google.com/maps/place/Your+Location/",
+        },
```

Comment:
The location pin can be a fixed navbar based on the `currentUser` status so we can just remove it from here.